src/pages/ShoppingCart/components/ShoppingCartHeader/ShoppingCartHeader.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../../../css/app.scss';
+
+.ShoppingCartHeader {
+  background-color: $secondary;
+  min-height: 130px;
+  max-height: 11.25rem;
```

Comment:
for max-* values maybe is better to use px anyway

Reply:
I agree and will make the change. Thanks!

Reply:
hmm thinking about it deeply, I think is better max-width in rem, since if the header has 180px of max height, and the user has set a bigger size basis, the most things wouldn't fit on the header 🤔, I think even the min height should be rem.

What do you think?